app/src/main/AndroidManifest.xml

```diff
@@ -46,6 +46,10 @@
         <meta-data
             android:name="com.google.android.gms.version"
             android:value="@integer/google_play_services_version" />
+
+        <!--<meta-data
+            android:name="com.google.android.geo.API_KEY"
+            android:value="AIzaSyD26n8P_N15DLoUP630EgBr-VvMdEu76U0"/>-->
```

Comment:
This is my key I used for testing, since our shared one doesn't have the places API enabled yet.